upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -469,8 +469,8 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		key := "networking.projectcalico.org"
 		versions := map[string]string{
 			"pre-k8s-1.6": "2.4.2-kops.1",
-			"k8s-1.6":     "2.6.7-kops.1",
-			"k8s-1.7":     "2.6.7-kops.1",
+			"k8s-1.6":     "2.6.8-kops.1",
```

Comment:
We just need to update the suffix here: (ie 2.6.7-kops.2)

Reply:
the reason that I bumped the version to 2.6.8 is that if I didn't, then existing clusters would not get the current deployment and daemonset updated

Reply:
Right but `2.6.8` would imply a different image which is not the case, just a config change, right? I agree with @robinpercy.  We shouldn't need to bump the version, just the `kops.1 -> kops.2` number

Reply:
oh, got it!